src/es/components/organisms/offersPage/OffersPage.js

```diff
@@ -370,24 +397,27 @@ export default class OffersPage extends Shadow() {
               <section>
                   <ks-m-sort namespace="sort-right-"></ks-m-sort>
               </section>
-              <ks-m-tile-factory></ks-m-tile-factory>
+              <ks-m-tile-factory ${this.eventDetailURL ? `is-event ` : ''}></ks-m-tile-factory>
               ${this.badgeContainer ? /* HTML */ `
                 <ks-m-badge-legend>
                   ${this.badgeContainer.innerHTML}
                 </ks-m-badge-legend>
               ` : ''}
-              <ks-a-with-facet-pagination>
-                <ks-a-button namespace="button-primary-" color="secondary">
-                    <span>Weitere Angebote</span>
-                    <a-icon-mdx namespace="icon-mdx-ks-" icon-name="ArrowDownRight" size="1em" class="icon-right">
-                </ks-a-button>
-              </ks-a-with-facet-pagination>
+              ${this.total && this.total > this.psize ? /* html */ `
```

Comment:
@Weedshaker Hier wird der Button nur gerendert, wenn ich mehr Ergebnisse als meine Page Size habe. In Zeile 13. setze ich die Werte an Hand der With Facet Response